ci/command_govet.go

```diff
@@ -0,0 +1,37 @@
+// +build mage
+
+/*
+ * Copyright (C) 2018 The "MysteriumNetwork/go-openvpn" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package main
+
+import (
+	"fmt"
+
+	"github.com/magefile/mage/sh"
+)
+
+// Checks that the source is compliant with go vet
+func GoVet() error {
+	out, err := sh.Output("go", "vet", "-composites=false", "../...")
```

Comment:
I'd like to be able to pass a path to project since we are, most probably, will be using it in node repo too.

Reply:
mage does not allow for arg support - they can only be passed as env variables, but I'll add an optional env variable